pallets/attesters/src/lib.rs

```diff
@@ -789,13 +1195,24 @@ pub mod attesters_test {
         });
     }
 
-    fn sign_and_submit_attestation(
+    fn sign_and_submit_sfx_attestation(
```

Comment:
I dont understand this function. The whole idea behind the batching, is that the batch is signed, not individual SFXs. What is this used for?